scope_registry.go

```diff
@@ -86,19 +87,30 @@ func (r *scopeRegistry) Subscope(parent *scope, prefix string, tags map[string]s
 		return NoopScope.(*scope)
 	}
 
-	key := scopeRegistryKey(prefix, parent.tags, tags)
-
+	buf := keyForPrefixedStringMapsAsKey(make([]byte, 0, 128), prefix, parent.tags, tags)
```

Comment:
IDs in practice can easily breach 256 bytes, buf of size 128 is going to escape quite often due to realloc in non-test cases.

Reply:
updated to 256. thanks.